OpenSubdiv/config.py

```diff
@@ -0,0 +1,37 @@
+{
+
+	"downloads" : [
+
+		"https://github.com/PixarAnimationStudios/OpenSubdiv/archive/v3_4_3.tar.gz"
+
+	],
+
+	"url" : "http://graphics.pixar.com/opensubdiv",
+
+	"license" : "LICENSE.txt",
+
+	"dependencies" : [ "TBB" ],
+
+	"commands" : [
+
+		"cmake"
+			" -D CMAKE_INSTALL_PREFIX={buildDir}"
+			" -D CMAKE_PREFIX_PATH={buildDir}"
+			" -D NO_GLFW=1"
```

Comment:
We may want ` -D NO_CUDA=1` here or ` -D OSD_CUDA_NVCC_FLAGS='--gpu-architecture compute_50'` or similar, as the defaults don't build on anything vaguely modern.

Reply:
https://github.com/PixarAnimationStudios/OpenSubdiv/issues/965

Reply:
Thanks - also fixed in 1777079. I've actually disabled a whole load more after discovering that they are disabled in the builds made by https://github.com/PixarAnimationStudios/USD/blob/release/build_scripts/build_usd.py. Note also one new commit, 99cca5e, which fixes a problem running `pyside2-uic` without having built Subprocess32 first.